src/app/projects/services/projects.service.ts

```diff
@@ -125,6 +134,13 @@ export class ProjectsService implements OnDestroy {
   }
 
   setFilters(filters: Filters) {
+    if (
+      filters.location &&
+      !ProjectsService.allowedLocations.includes(filters.location)
+    ) {
+      console.error(`${filters.location} not an allowed data location.`);
```

Comment:
Is that right that we only log the error to the console?

Reply:
Yes, I dont think this should be a fatal error that blocks the whole app

Reply:
I think we should be reporting app errors to a central location, like google analytics or Grafana, in order to be able to learn about app errors in the wild.